lib/src/components/Tab/tab.css.ts

```diff
@@ -0,0 +1,76 @@
+import { createTheme, style } from "@vanilla-extract/css";
+import { recipe } from "@vanilla-extract/recipes";
+import { TabTheme } from "./tab.types";
+
+export const [tabTheme, tabVars]: TabTheme = createTheme({
+  color: "#2F80ED",
+  background: "#BDDBFF60",
+  height: "50px",
+});
+
+export const tabHeaderContainerStyles = style({
+  height: tabVars.height,
+  position: "relative",
+  display: "flex",
+  alignItems: "center",
+  borderBottom: "2px solid #EBECF0",
+});
+
+export const tabRecipe = recipe({
+  base: {
+    cursor: "pointer",
+    height: "100%",
+    justifyContent: "center",
+    position: "relative",
+    zIndex: 1,
+    padding: "8px 10px",
+    display: "flex",
+    alignItems: "center",
```

Comment:
again could've used the `<Flex>` component here, but its fine for this version

Reply:
wokay